yarn-project/pxe/src/contract_function_simulator/oracle/utility_execution_oracle.ts

```diff
@@ -32,23 +32,27 @@ export class UtilityExecutionOracle extends TypedOracle {
     super();
   }
 
-  public override getBlockNumber(): Promise<number> {
+  public override utilityGetRandomField(): Fr {
```

Comment:
Added `utilityGetRandomField` implementation here because it used to be directly in `TypedOracle` and that goes against the standard usage of that class.